jobs/JGDAS_ATMOS_VERFOZN

```diff
@@ -15,29 +15,19 @@ export HOMEgfs_ozn=${HOMEgfs:-${NWROOT}/gfs.${gfs_ver}}
 export HOMEgdas_ozn=${HOMEgfs_ozn:-${NWROOT}/gfs.${gfs_ver}}
 export PARMgdas_ozn=${PARMgfs_ozn:-${HOMEgfs_ozn}/parm/mon}
 export SCRgdas_ozn=${SCRgfs_ozn:-${HOMEgfs_ozn}/scripts}
-export FIXgdas_ozn=${FIXgfs_ozn:-${HOMEgfs_ozn}/fix/gdas}
+export FIXgdas_ozn=${FIXgfs_ozn:-${HOMEgfs_ozn}/fix/mon}
 
 export HOMEoznmon=${HOMEoznmon:-${HOMEgfs_ozn}}
 export EXECoznmon=${EXECoznmon:-${HOMEoznmon}/exec}
 export FIXoznmon=${FIXoznmon:-${HOMEoznmon}/fix}
 export USHoznmon=${USHoznmon:-${HOMEoznmon}/ush}
 
-
-#-----------------------------------
-# source the parm file
-#
-.  ${PARMgdas_ozn}/gdas_oznmon.parm
-
-
 #############################################
 #  determine PDY and cyc for previous cycle
 #############################################
-
-pdate=$(${NDATE} -6 ${PDY}${cyc})
-echo "pdate = ${pdate}"
-
-export P_PDY=${pdate:0:8}
-export p_cyc=${pdate:8:2}
+GDATE=$(${NDATE} -"${assim_freq}" "${PDY}${cyc}")
```

Comment:
Rahul has been gradually changing these to use *nix `date` instead of NDATE:
```suggestion
GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
```